src/cloud/project/new-relic.md

```diff
@@ -250,13 +250,19 @@ For further information and examples, see [Introduction to New Relic Logs][New R
 
 ### Monitor performance with alert policies
 
-Adobe provides a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
+Adobe provides the Managed alerts for {{ site.data.var.ee }} alert policy on Cloud Pro and Starter Production environments to track the following metrics:
```

Comment:
```suggestion
Adobe provides the Managed alerts for {{ site.data.var.ee }} alert policy on Cloud Pro Production environments to track the following metrics:
```

Reply:
Can I also suggest that it either be "Managed Alerts" or "managed alerts"? As the two words are connected it looks odd having one uppercase and one lowercase